### ISSUE TITLE
REI Network: <Issue Title>

### ISSUE THREAD

**Issue by portussergey** [REI-Network/rei-scan#12]
Hi
My transaction doesn’t reach destination address, please advise the reason?

Txid: 
0x058d5ae04c543b4eb21766bf6aa208d8ef24a902c319121dc2a0776399da5ebc

Destination address:
0x080dCdC791A194E75BdB9b160953942162d0cC3D

Amount:  3968,814 REI

**Comment by Meta-Inu** [comment]
REI address

0x8b74Afa879C043bFc300991B6F34E4155E1B1375

**Comment by Meta-Inu** [comment]
https://scan.rei.network/tx/0x058d5ae04c543b4eb21766bf6aa208d8ef24a902c319121dc2a0776399da5ebc

**Comment by portussergey** [comment]
Correct, but final destination (0x080dCdC791A194E75BdB9b160953942162d0cC3D) not reached Brgds,Portus Sergiy Main ***@***.*** о 15:13 Meta-Inu ***@***.***> пише:﻿
https://scan.rei.network/tx/0x058d5ae04c543b4eb21766bf6aa208d8ef24a902c319121dc2a0776399da5ebc

**Comment by samlior** [comment]
Sorry for replying now, I found out that the transaction was packaged 10 days ago, any questions?

**Comment by portussergey** [comment]
Hi, I have same condition, transaction not reach final address.Brgds,Portus ***@***.*** о 15:34 Samlior ***@***.***> пише:﻿
Sorry for replying now, I found out that the transaction was packaged 10 days ago, any questions?

**Comment by samlior** [comment]
> Hi, I have same condition, transaction not reach final address.Brgds,Portus ***@***.*** о 15:34 Samlior ***@***.***> пише:﻿ Sorry for replying now, I found out that the transaction was packaged 10 days ago, any questions? —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

According to the information on the blockchain, the assets have already reached, and you can contact our customer service on telegram to further find out the problem https://t.me/ReiNetwork2021
